pocApp/new_tasks.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+from random import choice
+from celery import Celery
+from celery import shared_task, task
+from celery.decorators import periodic_task
+from libnmap.process import NmapProcess
+import xml.dom.minidom
+import sys
+import logging
+import datetime
+
+from pocApp.models import MyCertsData
+from orion_integration.lib import OrionSslNode
+
+logger = logging.Logger('catch_all')
+
+@shared_task
+def go():
+    return True
+    for i in range(10):
+        print ("I am @ record:%d " % (i))
+
+@task()
+def getnmapdata():
+    rc = "success"
+    nodeList = OrionSslNode.ip_addresses()
+    #nm = NmapProcess(param, options='--script ssl-cert')
+    for node in nodeList:
+        nm = NmapProcess(node, options='--script ssl-cert')
```

Comment:
you must catch nmap execution errors and you need to handle the case when nmap returns a filtered (firewalled) host, post tuple